pom.xml

```diff
@@ -57,7 +57,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>2.1.8</version>
+            <version>2.1.12</version>
```

Comment:
As a rule when updating a dep I would just go to the newest version unless there is a particular reason to hold back, but if you prefer 2.1.9 I could downgrade.